api/sql/method_edit_by_id.sql

```diff
@@ -1,17 +1,42 @@
 WITH full_thing AS (
   SELECT
-    methods.*,
-    COALESCE(images, '{}') AS images,
-    COALESCE(methods.files, '{}') files,
-    COALESCE(methods.videos, '{}') videos,
-    COALESCE(methods.tags, '{}') tags,
-    COALESCE(methods.links, '{}') links,
-    texts.body,
-    texts.title,
-    texts.description,
+    -- not user editable
+    id,
+    type,
+    original_language,
+    post_date,
+    published,
+    updated_date,
+    featured,
+    hidden,
     first_author(${articleid}) AS creator,
     last_author(${articleid}) AS last_updated_by,
-    bookmarked('method', ${articleid}, ${userid})
+    bookmarked('method', ${articleid}, ${userid}),
+    -- user-contributed content
+    texts.title,
+    texts.description,
+    texts.body,
+    -- media links
+    photos,
+    files,
+    videos,
+    links,
+    audio,
+    -- key values
+    facetoface_online_or_both,
+    public_spectrum,
+    open_limited,
+    recruitment_method,
+    level_polarization,
+    -- key lists
+    method_types,
+    number_of_participants,
+    scope_of_influence,
+    participants_interactions,
+    decision_methods,
+    if_voting,
+    -- yesno
+    facilitator
```

Comment:
i think the method spreadsheet uses `facilitator` but cases uses `facilitators`, should we use the plural for consistency? it uses the same list of options.

Reply:
I've been trying to use singular for things that take one result and plural for things that take multiple results, but obviously still haven't resolved all of that yet, and some fields it just doesn't work for (`audios`?). Ultimately it may not be that big of a deal, but would you prefer being consistent with cases by making it `facilitators` in methods, or by making it `facilitator` in cases?

Reply:
@ascott I've resolved all of the comments except for this one, and pushed the relevant changes. Once we resolve this we can merge this branch.

Reply:
Actually, we can merge now, I can normalize cases in a bugfix branch later.

Reply:
i'm fine with the plural or singular as long as they are the same across article types